test/cmd/edit.sh

```diff
@@ -22,5 +22,17 @@ os::cmd::expect_success_and_not_text 'OC_EDITOR=cat oc edit --windows-line-endin
 
 os::cmd::expect_success 'oc create -f test/testdata/services.yaml'
 os::cmd::expect_success_and_text 'OC_EDITOR=cat oc edit svc' 'kind: List'
+
+os::cmd::expect_success 'oc create imagestream test'
+os::cmd::expect_success 'oc create -f test/testdata/mysql-image-stream-mapping.yaml'
+os::cmd::expect_success_and_not_text 'oc get istag/test:new -o jsonpath={.metadata.annotations}' "tags:hidden"
+editorfile="$(mktemp -d)/tmp-editor.sh"
+echo '#!/bin/bash' > ${editorfile}
+echo 'sed -i "s/^tag: null/tag:\n  referencePolicy:\n    type: Source/g" $1' >> ${editorfile}
```

Comment:
@deads2k @mfojtik fyi, it would appear that @soltysh discovered the "you can't update imagestreamtags w/o explicitly adding a reference policy" issue back in february of 2017, but apparently no one cared then.

Reply:
I knew too much about image streams back then ;)